### ISSUE TITLE
Unable to share folders under Parallels

### ISSUE THREAD

**Issue by dazwin** [timsutton/osx-vm-templates#65]
- Box is based on 10.11.5 (El Capitan) (I followed https://spin.atomicobject.com/2015/11/17/vagrant-osx/ to the letter)
- Tried using nfs, as well as the default
- Vagrant messages seem to indicate the folders were mounted successfully...
```
    ==> default: Mounting shared folders...
    default: /vagrant => /Users/darrenw/Projects/####/#####
    default: /var/cache/apt/archives => /Users/darrenw/Projects/###/####/cache/apt/archives
```

The following interesting log entries are in /var/log/system.log:
````
com.apple.xpc.launchd[1] (com.parallels.vm.prl_fsd): Service only ran for 0 seconds. Pushing respawn out by 10 seconds.
````

...however, the process did start up by the time I logged in...
```
local:~ vagrant$ ps -ef | grep prl
    0    79     1   0  9:04PM ??         0:00.07 /Library/Parallels Guest Tools/prltoolsd
    0   181     1   0  9:04PM ??         0:00.06 /Library/Parallels Guest Tools/prl_fsd /Volumes/SharedFolders --share
```

If I open the GUI and try sharing a folder, it appears to work but, again, nothing actually gets shared. This might mean it's a Parallels issue, so I'll try installing directly (without osx-vm-templates) to see if that makes a difference.

**Comment by dazwin** [comment]
Sharing worked fine a regular Parallels/OSX guest. The problem is that when the VM is created, the 'Isolate Mac from virtual machine' option is turned on. Turning this off immediately exposes the shared folders.

Can this option be turned off when packer creates the image?

**Comment by timsutton** [comment]
Closing as this was fixed in #66.
